.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
Asserting that all comments are a code smell comes across as pretty dogmatic, and flagging them as lint errors is definitely not pragmatic.

(I'm for allowing comments)